scripts/ccf/propose.sh

```diff
@@ -3,23 +3,69 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+sign_proposal() {
+
+    proposal=$1
+
+    if [[ $USE_AKV == false ]]; then
+        ccf_cose_sign1 \
+            --content $proposal \
+            --signing-cert ${KMS_MEMBER_CERT_PATH} \
+            --signing-key ${KMS_MEMBER_PRIVK_PATH} \
```

Comment:
you should use ccf_cose_sign1_prepare/finish to sign on AKV.
See sign_proposal_akv.sh

Reply:
This code is if `USE_AKV == false` please see the else case three lines down